src/main/java/net/fabricmc/loom/LoomGradleExtension.java

```diff
@@ -58,6 +62,7 @@
 import net.fabricmc.loom.configuration.providers.minecraft.MinecraftMappedProvider;
 
 public class LoomGradleExtension {
+	@Deprecated // Configured in run configurations now
 	public String runDir = "run";
```

Comment:
Does this still work? Generally `@Deprecated` stuff should work as usual, but I didn't see any compat code hooking this up to the new system. (If it doesn't work, it should be removed instead of confusing users)

Reply:
No, good point. I was just going to remove it. I don’t think it was commonly used anyway.

Reply:
Seems to still be used to generate the directories, will full remove and fix.